controllers/csiaddons/reclaimspacejob_controller.go

```diff
@@ -324,7 +324,7 @@ func (r *ReclaimSpaceJobReconciler) getTargetDetails(
 		pvName:     pv.Name,
 	}
 	for _, v := range volumeAttachments.Items {
-		if *v.Spec.Source.PersistentVolumeName == pv.Name {
+		if v.DeletionTimestamp.IsZero() && v.Status.Attached && *v.Spec.Source.PersistentVolumeName == pv.Name {
```

Comment:
This is not true for controller space reclaim, as PV is not required to be attached, right?

Reply:
> This is not true for controller space reclaim, as PV is not required to be attached, right?

We don't need node information for performing controller space reclaim.
It will still happen.